auth-web/src/components/auth/manage-business/manage-business-dialog/AccountAuthorizationRequest.vue

```diff
@@ -40,19 +40,22 @@
         aria-label="Request access additional message"
         maxlength="400"
         counter="400"
-        @change="$emit('changeRequestAccessMessage', componentState.requestAccessMessage)"
+        @change="$emit('change-request-access-message', componentState.requestAccessMessage)"
       />
     </div>
-    <div v-else>
-      <h3 class="text-center">
-        No authorizing accounts found
-      </h3>
+    <div v-else id="no-accounts-found">
+      <p>An error occurred retrieving the list of authorizing accounts for this business.</p>
```

Comment:
I think a common component for this will make sense, we're doing this for the 4th option. Relationships can refactor this